Source/Forms/Gen8/WildPLA.cpp

```diff
@@ -0,0 +1,156 @@
+#include "WildPLA.hpp"
+#include "ui_WildPLA.h"
+
+#include <Core/Gen8/Generators/WildGeneratorPLA.hpp>
+#include <Core/Gen8/Profile8.hpp>
+#include <Core/Parents/ProfileLoader.hpp>
+#include <Forms/Controls/Controls.hpp>
+#include <Forms/Gen8/Profile/ProfileManager8.hpp>
+#include <Forms/Models/Gen8/RaidModel.hpp>
+#include <QMenu>
+#include <QSettings>
+
+WildPLA::WildPLA(QWidget *parent) : QWidget(parent), ui(new Ui::WildPLA)
+{
+    ui->setupUi(this);
+
+    setAttribute(Qt::WA_QuitOnClose, false);
+
+    updateProfiles();
+    setupModels();
+}
+
+WildPLA::~WildPLA()
+{
+    QSettings setting;
+    setting.beginGroup("WildPLA");
+    setting.setValue("profile", ui->comboBoxProfiles->currentIndex());
+    setting.setValue("geometry", this->saveGeometry());
+    setting.endGroup();
+
+    delete ui;
+}
+
+bool WildPLA::hasProfiles() const
+{
+    return !profiles.empty();
+}
+
+void WildPLA::updateProfiles()
+{
+    profiles.clear();
+    auto completeProfiles = ProfileLoader8::getProfiles();
+    std::copy_if(completeProfiles.begin(), completeProfiles.end(), std::back_inserter(profiles),
+                 [](const Profile &profile) { return profile.getVersion() == Game::PAL; });
+    profiles.insert(profiles.begin(), Profile8(Game::PAL));
+
+    ui->comboBoxProfiles->clear();
+
+    for (const auto &profile : profiles)
+    {
+        ui->comboBoxProfiles->addItem(QString::fromStdString(profile.getName()));
+    }
+
+    QSettings setting;
+    int val = setting.value("WildPLA/profile", 0).toInt();
+    if (val < ui->comboBoxProfiles->count())
+    {
+        ui->comboBoxProfiles->setCurrentIndex(val);
+    }
+
+    profilesIndexChanged(0);
```

Comment:
I don't think we want this here

Reply:
Was here from copying over from another form as a template, will remove.